Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -116,6 +127,14 @@ void UPSWorldSubsystem::RegisterSpotComponent(UPSSpotComponent* MySpotComponent)
 		return;
 	}
 	PSSpotComponentArrayInternal.AddUnique(MySpotComponent);
+
+	for (TTuple<FName, FPSRowData> RowData : ProgressionSettingsDataInternal)
```

Comment:
TTuple is complex data type, it has to be as const-ref, otherwise you copy the value on each iteration: `for (const TTuple<FName, FPSRowData>& RowData : ProgressionSettingsDataInternal)`

Reply:
@moinm3uw

Reply:
removed as part of https://github.com/moinm3uw/ProgressionSystem/pull/59/commits/94927e7d26f018f9cd0d9e3c503f1ce8471c21d0